appenders/python/pulse/formatters.py

```diff
@@ -26,5 +27,11 @@ def format(self, record):
         data["level"] = record.levelname
         data["message"] = record.msg
         data["threadName"] = record.threadName
+        try:
+            data["hostname"] = socket.gethostname()
+            if not data["hostname"]:
+                raise RuntimeError('No HostName received via socket')
```

Comment:
This shouldn't be a runtime error. Is there a logging/debug mechanism in the logger? We don't want to fail silently, but it would be nice to be able to log a message by setting `export PULSE_DEBUG=true`. If this doesn't exist yet just swallow the exception.